requirements.txt

```diff
@@ -1 +1 @@
-pytest==7.2.1
\ No newline at end of file
+pytest==7.4.3
```

Comment:
Depending on the size of such changes and the practices used by the team I would consider making this a separate PR. Given the somewhat artificial nature of this exercise and the small size of the change in this case, having this as an first commit in the PR seems sufficient.